src/state/ducks/userDuck.js

```diff
@@ -24,6 +24,7 @@ export const userActions = {
     axiosAuth()
       .get('/users/getuserinfo')
       .then(res => {
+        console.log(res.data);
```

Comment:
Hi Chris, could you please remove this console.log from the code. That way it's nice and clean for production deploy!

Reply:
Done. Sorry about that. Yeah, this seemed like the most rational decision.